pkg/types/validation/installconfig.go

```diff
@@ -163,6 +164,22 @@ func ipnetworksToStrings(networks []ipnet.IPNet) []string {
 	return diag
 }
 
+// returns true if the plugin uses the hostPrefix field or
```

Comment:
this functional comment does seem to follow the go guidelines see https://blog.golang.org/godoc

Reply:
my bad, I should have paid more attention. will fix this.